cfe_internal/update/update_policy.cf

```diff
@@ -236,16 +236,18 @@ bundle agent cfe_internal_update_policy_cpv
       action => u_immediate,
       classes => u_if_repaired("validated_updates_ready");
 
-    !am_policy_hub::
+    any::
 
       "$(sys.workdir)$(const.dirsep)modules"
-      comment => "Always update modules files on client side",
+      comment => "Update modules from masterfiles",
       handle => "cfe_internal_update_policy_files_update_modules",
-      copy_from => u_rcp("$(modules_dir_source)", @(update_def.policy_servers)),
+      copy_from => u_cp("$(master_location)$(const.dirsep)modules"),
       depth_search => u_recurse("inf"),
       perms => u_m("755"),
```

Comment:
Will this be the right perms on windows?

Reply:
from https://docs.cfengine.com/docs/master/reference-promise-types-files.html#mode:

> Notes: This is ignored on Windows, as the permission model uses ACLs.

Also note that Windows doesn't have "executable" permission bit, but checks file extensions instead. So if you can read an `*.exe` file - you can as well execute it.